android/expoview/src/main/java/host/exp/exponent/experience/splashscreen/ManagedAppSplashScreenConfiguration.kt

```diff
@@ -69,8 +70,30 @@ class ManagedAppSplashScreenConfiguration private constructor() {
       } else null
     }
 
+    /**
+     * Tries to retrieve imageUrl from the manifest checking for value for keys/paths in following order
+     * - android-scoped splash dpi images (starting from 'xxx-hdpi" and ending with 'mdpi')
+     * - android-scoped splash imageUrl
+     * - generic splash imageUrl
+     */
     private fun parseImageUrl(manifest: JSONObject): String? {
-      return getStringFromManifest(
+      val androidSplash = manifest
+        .optJSONObject(ExponentManifest.MANIFEST_ANDROID_INFO_KEY)
+        ?.optJSONObject(ExponentManifest.MANIFEST_SPLASH_INFO_KEY)
+      if (androidSplash != null) {
+        val dpiRelatedImageUrl = getStringFromJSONObject(
```

Comment:
So… would this return the first non-empty value for any of the keys defined in `*arrayOf("xxxhdpi"…)`? Wouldn't we want to use a lower-resolution image on a lower-resolution device?

Reply:
We've been always doing this that way (code from the `sdk-38` branch): https://github.com/expo/expo/blob/9db4ea6dcb8431faac64047f7580cdaa1c8d6df0/android/expoview/src/main/java/host/exp/exponent/LoadingView.java#L271-L295

The image is fetched into `ImageView`, so we rather prefer to always have the best resolution here.
These `*dpi` are only usable in standalone's builds.

Reply:
Ah so this won't be used in standalone apps, sorry. 👍

Reply:
if it's the way we've always done it then let's stick with that. we can follow up with improvements over status quo if needed in another pr